app/views/story/_stories.html.erb

```diff
@@ -3,6 +3,6 @@
                     method: :get %></b>| Date: <%= story.created_at %><br>
 	<%= story.part_of_the_content %><br>
 	Author: <%= story.user.full_name %><br>
+	<%= raw story.tags.map { |t| link_to t.name, tag_path(t) }.join(', ') %>
```

Comment:
why raw here?

Reply:
If I ommit it will not be rendered in form of links, just in html.

Reply:
I replaced it with html_safe in story decorator. It is doing the same